README.md

```diff
@@ -27,16 +27,23 @@ By default, we collect all we can find. You can control this by only enabling ce
-|       | --nodetails       | Disable detailed collection including logs and more                                                                    |
-|       | --enable          | List of enabled modules (default: all)                                                                                 |
-|       | --disable         | List of disabled modules (default: none)                                                                               |
-|       | --hide            | List of keywords to obfuscate. Can be used multiple times                                                              |
-|       | --command-timeout | Timeout for command execution in modules (default: 1m0s)                                                               |
-|  -v   | --verbose         | Enable verbose logging                                                                                                 |
-|  -V   | --version         | Print version and exit                                                                                                 |
+To collect advanced data for module `Icinga 2`, you can use the Icinga 2 API to collect data from all endpoints provided.
+The API requests are performed with a global API user you have to create yourself. Just create that user in a global zone like 'director-global'
+
+
+| Short | Long                    | Description                                                                                                                                   |
+|:-----:|:------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+|  -o   | --output                | Output file for the zip content (default: current directory and named like '$HOSTNAME'-netways-support-$TIMESTAMP.zip)                        |
+|       | --nodetails             | Disable detailed collection including logs and more                                                                                           |
+|       | --enable                | List of enabled modules (default: all)                                                                                                        |
+|       | --disable               | List of disabled modules (default: none)                                                                                                      |
+|       | --hide                  | List of keywords to obfuscate. Can be used multiple times                                                                                     |
+|       | --command-timeout       | Timeout for command execution in modules (default: 1m0s)                                                                                      |
+|       | --icinga2-api-user      | Username of global Icinga 2 API user to collect data about Icinga 2 Infrastructure                                                            |
```

Comment:
Been thinking, maybe we don't want to put application specifics into the command line interface, since that might lead to more and more application specifics.

A more longterm solution might be a configuration file for the tool, that can be specified with `--config`.

A YAML/JSON config could look like this:

```
appxy:
  endpoint: "localhost"
  user: "admin"
  pass: "secret123"

app2:
  user: "admin"
  pass: "secret123"
  cert: /foo.cert
  key: /foo.key
```

This could also then be used to pass connection specific configuration, such as TLS. Otherwise the CLI becomes unwieldy

Reply:
Especially if we want to move to a stable 1.0.0, I think this might be a good move

Reply:
I already had similar ideas that I would like to implement.
However, as described in #111 , I don't see a conf file here as being quite as useful for the use case we want to cover.

Such a feature should be tackled independently.